test/cypress/integration/edit_mode_spec.js

```diff
@@ -257,14 +256,14 @@ describe('Edit mode', () => {
             const widgetId = `testWidget${widgetName}`;
             const url = 'https://this-page-intentionally-left-blank.org/';
             const verifyRequest = service =>
-                cy.wait(service).then(xhr => {
-                    expect(xhr.requestHeaders).contain({ 'widget-id': widgetId });
-                    expect(xhr.status).equals(200);
+                cy.wait(service).then(({ request, response }) => {
+                    expect(request.headers).contain({ 'widget-id': widgetId });
+                    expect(response.statusCode).equals(200);
                 });
             const verifyResponse = text => cy.get('.widgetContent .ui.segment pre code').should('contain.text', text);
 
-            cy.route('GET', '/console/wb/manager?endpoint=version').as('managerService');
-            cy.route('GET', `/console/wb/request?url=${url}`).as('requestService');
+            cy.intercept('GET', '/console/wb/manager?endpoint=version').as('managerService');
```

Comment:
Here probably you can use:
```suggestion
            cy.intercept({ method: 'GET', pathname: '/console/wb/manager', query: { endpoint: 'version' }}).as('managerService');
```